db/migrate/20200510154151_add_reference_id_to_conversation.rb

```diff
@@ -0,0 +1,5 @@
+class AddReferenceIdToConversation < ActiveRecord::Migration[6.0]
+  def change
+    add_column :conversations, :reference_id, :string
```

Comment:
@subintp  since this attribute will only be used in only x % of conversations lets make use of the ` additional_attributes`  jsonb column to store the reference id.  probably let's name it as 'slack_reference_id` to avoid confusion in the future.

Reply:
@sojan-official  Please check the below comment.